tuid_provider/views.py

```diff
@@ -0,0 +1,13 @@
+from allauth_cas.views import CASAdapter, CASCallbackView, CASLoginView, CASLogoutView
+from .provider import TUIDProvider
+
+
+class TUIDAdapter(CASAdapter):
+    provider_id = TUIDProvider.id
+    url = 'https://sso.tu-darmstadt.de'
```

Comment:
Should this url be hardcoded?

Reply:
Oh yes I better should write some default url mechanism similar to the d120 provider.

Reply:
fixed with 4ed60454f0d159143a94a783cdbe262032beb273